app/manage/organizations.js

```diff
@@ -0,0 +1,221 @@
+const organization = require('../lib/organization')
+const team = require('../lib/team')
+
+/**
+ * Create an organization
+ * Uses the user id in the request and the body to forward
+ * to the organization model
+ */
```

Comment:
We have not done this for the other routes, because they are standard express objects. Do you think we should document the expected request / reply jsdocs?

Reply:
In scrum meeting we decided to skip this because the jsdoc would be too repetitive.